pkg/synthetictests/historicaldata/next_best_guess.go

```diff
@@ -0,0 +1,163 @@
+package historicaldata
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/openshift/origin/pkg/synthetictests/platformidentification"
+)
+
+// nextBestGuessers is the order in which to attempt to lookup other alternative matches that are close to this job type.
+var nextBestGuessers = []NextBestKey{
+	MicroReleaseUpgrade,
+	MinorReleaseUpgrade,
+	PreviousReleaseUpgrade,
+	combine(PreviousReleaseUpgrade, MicroReleaseUpgrade),
+	combine(PreviousReleaseUpgrade, MinorReleaseUpgrade),
+	OnOVN,
+	OnSDN,
+	combine(PreviousReleaseUpgrade, OnOVN),
+	combine(PreviousReleaseUpgrade, OnSDN),
+	combine(PreviousReleaseUpgrade, MicroReleaseUpgrade, OnOVN),
+	combine(PreviousReleaseUpgrade, MicroReleaseUpgrade, OnSDN),
+	combine(PreviousReleaseUpgrade, MinorReleaseUpgrade, OnOVN),
+	combine(PreviousReleaseUpgrade, MinorReleaseUpgrade, OnSDN),
+}
+
+/*
+  {
+    "BackendName": "ingress-to-oauth-server-reused-connections",
+    "Release": "4.11",
+    "FromRelease": "4.10",
+    "Platform": "azure",
+    "Network": "ovn",
+    "Topology": "ha",
+    "P95": "3.8499999999999996",
+    "P99": "3.9699999999999998"
+  },
+  {
+    "BackendName": "ingress-to-oauth-server-reused-connections",
+    "Release": "4.11",
+    "FromRelease": "4.10",
+    "Platform": "azure",
+    "Network": "sdn",
+    "Topology": "ha",
+    "P95": "0.0",
+    "P99": "12.869999999999884"
+  },
+
+*/
+
+// NextBestKey returns the next best key and a bool indicating whether this guesser has an opinion.
```

Comment:
"next best key in the query_results.json generated from BigQuery, and a bool..."